flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -1057,19 +1057,22 @@ class CDUPerformancePage {
             SimVar.SetSimVarValue("L:AIRLINER_ACC_ALT", "Number", accAlt);
         }
     }
-    static UpdateEngOutAccFromOrigin(mcdu) {
-        if (mcdu.engineOutAccelerationAltitudeIsPilotEntered) {
-            return;
-        }
```

Comment:
Can we keep this here? I think it's a good idea to have this check at the start of the function to prevent a few unnecessary calls below.

Reply:
> Can we keep this here? I think it's a good idea to have this check at the start of the function to prevent a few unnecessary calls below.

Yes I agree a quick exit from the function seems a good idea. That would imply it would be good to add the same a similar exit for THR RED / ACC.

Reply:
if (mcdu.engineOutAccelerationAltitudeIsPilotEntered) {
            return;
        }
Amended to include above code.